docs-client/src/containers/MethodPage/DebugPage.tsx

```diff
@@ -179,10 +179,15 @@ const DebugPage: React.FunctionComponent<Props> = ({
     const urlParams = new URLSearchParams(location.search);
 
     let urlRequestBody = '';
+    let urlDebugFormIsOpen = false;
     if (useRequestBody) {
       if (urlParams.has('request_body')) {
         urlRequestBody = jsonPrettify(urlParams.get('request_body')!);
       }
+
+      if (urlParams.has('debug_form_is_open')) {
```

Comment:
Should we open a debug form when `debug_form_is_open` is set to `true` regardless of `useRequestBody`?

Reply:
Yes